app/models/custom_field.rb

```diff
@@ -80,12 +86,17 @@ def self.loan_questions(field_set = nil)
   # - If override is false, inherit from parent
   # Note, loan type association records are ignored for questions without the 'override_assocations'
   # flag assigned.
+  # def required_for?(loan)
+  #   if override_associations
+  #     loan_types.include?(loan.loan_type_option)
+  #   else
+  #     parent && parent.required_for?(loan)
+  #   end
+  # end
+
+  # Temporary stub to demonstrate functionality
   def required_for?(loan)
-    if override_associations
-      loan_types.include?(loan.loan_type_option)
-    else
-      parent && parent.required_for?(loan)
-    end
+    id % 2 == 1
```

Comment:
Did you see the comment, "Temporary stub to demonstrate functionality"? There's not yet a way for the user to set questions as required. This is a consistent random sampling.